data/korath/korath ships.txt

```diff
@@ -84,8 +84,8 @@ ship "Rano'erek"
 	explode "large explosion" 50
 	explode "huge explosion" 15
 	"final explode" "final explosion large"
-	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and raiders lacked enough cargo capacity. The Rano'erek-class Asteroid Dredging Ship, named for a respected civilian politician and economist, was born as an answer to this problem."
-	description "	Featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Palavrets."
+	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and raiders lacked the cargo capacity. The Rano'erek-class Asteroid Dredging Ship, named for a respected civilian politician and economist, was born as an answer to this problem."
+	description "	Featuring an abundance of cargo and passenger space, and formidable defensive armament, these bulky ships can also offer potent support for frontline Palavrets."
```

Comment:
```suggestion
	description "	Featuring an abundance of cargo and passenger space and formidable defensive armament, these bulky ships can also offer potent support for frontline Palavrets."
```

Reply:
The comma feels necessary given the other and earlier in the sentence.  As written,
"cargo and passenger space" is abundant
"defensive armament" is formidable, not necessarily abundant

While it may or may not be correct to say the defensive armament is also abundant, it makes the sentence clunky with double ands